crates/gpui/Cargo.toml

```diff
@@ -72,6 +72,7 @@ usvg = { version = "0.14", features = [] }
 util.workspace = true
 uuid = { version = "1.1.2", features = ["v4"] }
 waker-fn = "1.1.0"
+copypasta = "0.10.1"
```

Comment:
This should be moved to the `[target.'cfg(any(target_os = "linux", target_os = "windows"))'.dependencies]` section.